packages/mediacenter/kodi/package.mk

```diff
@@ -229,14 +229,13 @@ export CXXFLAGS_FOR_BUILD="$HOST_CXXFLAGS"
 export CFLAGS_FOR_BUILD="$HOST_CFLAGS"
 export LDFLAGS_FOR_BUILD="$HOST_LDFLAGS"
 
-export PYTHON_VERSION="2.7"
```

Comment:
Yes, we need to reinstate `export PYTHON_VERSION="2.7"` - thanks @anaconda.

Seems odd to have removed it then hard-code 2.7 in multiple places. @lrusak can we change this back so that wherever there is a reference to 2.7 it uses $PYTHON_VERSION instead?